nexus/db-model/src/console_session.rs

```diff
@@ -4,13 +4,18 @@
 
 use chrono::{DateTime, Utc};
 use nexus_db_schema::schema::console_session;
+use omicron_uuid_kinds::ConsoleSessionKind;
+use omicron_uuid_kinds::TypedUuid;
```

Comment:
similarly, since this file must reference the kind type directly in `DbTypedUuid`, i'm fine with not using the alias.